src/components/candidate-form.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import axios from "axios";
+
+export default function CandidateForm(props) {
+  const { name, voteCount, rank, id } = props;
+
+  const voteCandidate = () => {
+    axios
+      .put(process.env.API_HOST + "/candidates/" + id + "/vote/", {
+        params: {},
+      })
```

Comment:
1. 템플릿 리터럴이라는 문법이 존재합니다. 백틱(`)을 이용해서 문자열을 감싸주는 방법입니다. 이 문법의 장점은 백틱 안에 표현식을 넣을 수 있다는 점입니다. 그래서 아래와 같이 수정할 수 있을 거 같아요!. 자세한 내용은 밑에 링크를 참고해주세요.

- [ES6 템플릿 리터럴](https://eblee-repo.tistory.com/38)

2. put요청 시 파라미터가 없다면, 생략해도됩니다.

```suggestion
      .put(process.env.API_HOST + `/candidates/${id}/vote/`)
```